symcrypt-sys/src/bindings/x86_64_unknown_linux_gnu.rs

```diff
@@ -4198,6 +4198,11 @@ extern "C" {
 extern "C" {
     pub fn SymCryptStoreMsbFirstUint64(src: UINT64, pbDst: PBYTE, cbDst: SIZE_T) -> SYMCRYPT_ERROR;
 }
+#[cfg(not(feature = "dynamic"))]
```

Comment:
`SymCryptModuleInit()` is only exposed when `dynamic` feature is present. This applies for all triples